detections/cloud/o365_add_app_role_assignment_grant_user.yml

```diff
@@ -14,11 +14,11 @@ description: The following analytic detects the addition of an application role
   access to critical resources and data within the Office 365 environment.
 data_source:
 - O365 Add app role assignment grant to user.
-search: '`o365_management_activity` Workload=AzureActiveDirectory Operation="Add app
-  role assignment grant to user." | stats count min(_time) as firstTime max(_time)
-  as lastTime values(Actor{}.ID) as Actor.ID values(Actor{}.Type) as Actor.Type values(ModifiedProperties{}.Name)
-  as modified_properties_name by user dest ResultStatus Operation | `security_content_ctime(firstTime)`
-  | `security_content_ctime(lastTime)` | `o365_add_app_role_assignment_grant_user_filter`'
+search: '`o365_management_activity` Workload=AzureActiveDirectory Operation="Add app role assignment grant to user."
+  | rename command as action
```

Comment:
Similar to azure ad detections, lets not rename `| rename command as action` instead use `signature` in the output field for all the detections as a mandatory output field. It should be present in all events

Reply:
This is fixed